Userland/Utilities/du.cpp

```diff
@@ -30,28 +30,29 @@ struct DuOption {
     bool all = false;
     bool apparent_size = false;
     int threshold = 0;
+    int max_depth = INT_MAX;
     TimeType time_type = TimeType::NotUsed;
     Vector<String> excluded_patterns;
 };
 
-static ErrorOr<void> parse_args(Main::Arguments arguments, Vector<String>& files, DuOption& du_option, int& max_depth);
-static ErrorOr<off_t> print_space_usage(const String& path, const DuOption& du_option, int max_depth, bool inside_dir = false);
+static ErrorOr<void> parse_args(Main::Arguments arguments, Vector<String>& files, DuOption& du_option);
+static ErrorOr<off_t> print_space_usage(const String& path, const DuOption& du_option, int depth_level, bool inside_dir = false);
 
 ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
     Vector<String> files;
     DuOption du_option;
-    int max_depth = INT_MAX;
+    int depth_level = 0;
 
-    TRY(parse_args(arguments, files, du_option, max_depth));
+    TRY(parse_args(arguments, files, du_option));
 
     for (const auto& file : files)
-        TRY(print_space_usage(file, du_option, max_depth));
+        TRY(print_space_usage(file, du_option, depth_level));
```

Comment:
You can at least get rid of the `depth_level` variable.
And I would suggest to make `print_space_usage` have a default value of `0` for `depth_level`.

Reply:
Good suggestion. I've removed `depth_level` from main and added a default argument for `depth_level` in `print_space_usage`.